src/views/socialAuthView/socialAuthView.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import toastr from 'toastr';
+import { withRouter } from 'react-router-dom';
+import SocialAuth from '../../components/socialAuth/socialAuth';
+import googleSignInAction from '../../actions/socialActions/googleAction';
+import facebookSignInAction from '../../actions/socialActions/facebookAction';
+
+
+export class SocialAuthView extends Component {
+    handleFacebookReponse=(response) => {
+      const { facebookSignInAction } = this.props;
+      if (response.accessToken) {
+        facebookSignInAction(response.accessToken);
+      }
+    }
+
+    handleGoogleResponseSuccess=(response) => {
+      const { googleSignInAction } = this.props;
+      if (response.tokenId) {
+        googleSignInAction(response.tokenId);
+      }
+    }
+
+    handlegoogleResponseFailure=(response) => {
+      toastr.warning(response.error);
+      const { googleSignInAction } = this.props;
+      googleSignInAction('invalid request');
+    }
+
+    componentWillReceiveProps(nextprops) {
+      if (nextprops.socialAuthState.isAuthenticated) {
+        toastr.info('Login successful');
```

Comment:
@MuhweziDeo Please consider  adding the username of the logged in user to the success message.

Reply:
cool